alica_engine/include/engine/IAssignment.h

```diff
@@ -23,12 +22,13 @@ class AssignmentCollection;
 /**
  *  An IAssignment describes a potentially partial assignment of robots to EntryPoints within a plan.
  */
-class IAssignment {
-public:
+class IAssignment
+{
```

Comment:
I thought that alica developers also settled on brackets in same line?

Reply:
Erm, no, this is the compromise...
brackets on the same line only for blocks